readme.md

```diff
@@ -58,6 +58,77 @@ Values: `black` `red` `green` `yellow` `blue` `magenta` `cyan` `white` `gray`
 
 Color of the box border.
 
+##### borderStyle
+
+Type: `string`, `object`
+Default: `single`
+Values:
+- `single`, e.g.
+  ```
+┌───┐
+│foo│
+└───┘
+  ```
+- `double`
+```
+╔═══╗
+║foo║
+╚═══╝
+```
+- `round` (`single` sides with round corners)
+```
+╭───╮
+│foo│
+╰───╯
+```
+- `single-double` (`single` on top and bottom, `double` on right and left)
+```
+╓───╖
+║foo║
+╙───╜
+```
+- `single-double-single-double` (alias for `single-double`)
+```
+╓───╖
+║foo║
+╙───╜
+```
+- `double-single` (`double` on top and bottom, `single` on right and left)
+```
+╒═══╕
+│foo│
+╘═══╛
+```
+- `double-single-double-single` (alias for `double-single`)
+```
+╒═══╕
+│foo│
+╘═══╛
+```
+
+Style of the box border. Can be any of the predefined styles from above or an object with the following keys:
+
+- `tl`: The string to use for the top-left corner
+- `tr`: The string to use for the top-right corner
+- `bl`: The string to use for the bottom-left corner
+- `br`: The string to use for the bottom-right corner
```

Comment:
I know it's more verbose, but the current ones are too obscure, can you use full vars? `topLeft`, `topRight`, `bottomLeft`, `bottomRight`.